src/main/scala/mesosphere/util/state/MesosLeaderInfo.scala

```diff
@@ -0,0 +1,30 @@
+package mesosphere.util.state
+
+/**
+  * Utility class for keeping track of a Mesos Leader
+  */
+trait MesosLeaderInfo {
+
+  def currentLeaderUrl: Option[String]
```

Comment:
Maybe document which URL it is exactly with an example. Does it include a trailing slash? etc.